src/main/java/uk/gov/justice/digital/domains/service/DomainService.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.domains.service;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+
+import uk.gov.justice.digital.domains.model.TableInfo;
+import uk.gov.justice.digital.service.DeltaLakeService;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public class DomainService {
+
+    protected DeltaLakeService deltaService = new DeltaLakeService();
```

Comment:
Use micronaut to inject dependencies. The datahub code and the code it uses is doing this so take a look at the code there. You'd typically want to declare the delta lake service as a singleton.